react/Installments.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { Installment, InstallmentOption } from 'vtex.checkout-graphql'
+import { useIntl, defineMessages } from 'react-intl'
+import { useFormattedPrice } from 'vtex.formatted-price'
+import { ListGroup, GroupOption } from 'vtex.checkout-components'
+import { useOrderPayment } from 'vtex.order-payment/OrderPayment'
+
+import CardSummary from './CardSummary'
+import Header from './components/Header'
+import { PaymentType } from './enums/PaymentEnums'
+
+const messages = defineMessages({
+  installmentValue: {
+    id: 'store/checkout-payment.installmentValue',
+  },
+  interestFree: {
+    id: 'store/checkout-payment.interestFree',
+  },
+  paymentOptionLabel: {
+    id: 'store/checkout-payment.paymentOptionLabel',
+  },
+})
+
+const InstallmentItem: React.FC<{
+  installment: Installment
+}> = ({ installment }) => {
+  const intl = useIntl()
+
+  const formattedPrice = useFormattedPrice(installment!.value! / 100)
+
+  const installmentValue = intl.formatMessage(messages.installmentValue, {
+    installments: installment.count,
+    value: formattedPrice,
+  })
+
+  const interestRate =
+    installment.hasInterestRate || installment.count === 1
+      ? ''
+      : intl.formatMessage(messages.interestFree)
+
+  return (
+    <>
+      <div className="flex-auto">{installmentValue}</div>
+      <div className="flex-none c-success">{interestRate}</div>
```

Comment:
Although correct, i'm not sure that `c-success` is very semantic to display something that is not a success result.

Reply:
you're right, do you have any suggestions to which class I should use here? I think the price is also using this class (when the price is 0 and the label "FREE" is displayed)